spec/jobs/notify_subscription_confirmation_email_job_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe NotifySubscriptionConfirmationEmailJob, type: :job do
+  include ActiveJob::TestHelper
+
+  subject(:job) { described_class.perform_later(subscription.id) }
+  let(:subscription) { create(:daily_subscription) }
+
+  it 'queues the job' do
+    expect { job }.to change(ActiveJob::Base.queue_adapter.enqueued_jobs, :size).by(1)
+  end
+
+  it 'is in the notify_request  queue' do
+    expect(job.queue_name).to eq('notify_request')
+  end
+
+  it 'writes to the spreadsheet' do
```

Comment:
Is this the right description for the test? It looks to me like this asserts the email service is invoked, rather than a spreadsheet job is enqueued. Perhaps this is down to copy and paste since we have a fair number of tests that write to spreadsheets.